### PR TITLE
Add support for statusCallback

### PR DESCRIPTION
**Pull request by jraede**
@raix please review when you get a minute - this is needed for the new notifications project

### CODE REVIEW

File: twilio.js [hunk 1]
```diff
@@ -49,18 +52,22 @@ Twilio.prototype.sendSMS = function (options, callback) {
  * {String} options.body
  * {String} options.mediaUrl
```
Comment (jperl) [inline twilio.js]: Not something you did --- but this should be `[options.mediaUrl]` since it is optional

**Comment by jperl** [comment]
LGTM :+1: Merge it!

File: twilio.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ var twilio = Npm.require('twilio');
 
 var lookups = new Mongo.Collection('twilio.lookups');
 
-Twilio = function(options) {
+Twilio = function (options) {
```
Comment (raix) [inline twilio.js]: is it just me or does it look odd "naming" the function with a space?

Comment (jraede) [inline twilio.js]: That's the "official" Meteor jsbeautify rules, no? I don't have a preference either way, my editor just auto formats it according to the `.jsbeautifyrc` file

**Pull request merged**